src/test/java/edu/kis/powp/jobs2d/events/SelectChangeVisibleOptionListener.java

```diff
@@ -14,10 +14,6 @@ public SelectChangeVisibleOptionListener(Window controlledWindow) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		if (controlledWindow.isVisible()) {
-			controlledWindow.setVisible(false);
-		} else {
-			controlledWindow.setVisible(true);
-		}
+        controlledWindow.setVisible(!controlledWindow.isVisible());
 	}
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.